### PR TITLE
chore(finance): Migrate kafka topics debt payment page #899 [DEV]

### PR DESCRIPTION
**Pull request by victoriaUW**
This PR is a breakdown of #750 

https://linear.app/utilitywarehouse/issue/FIN-899/migrate-topics-related-to-debt-payments-page

Created a new tf file for the debt payment page

- bq-staged-events
- bq-deadletter-events
- debt-payment-transaction
- overdue-balance-with-payments-diffs
- failed-payment-confirmation

Checked configurations with original kubernetes manifesto

### CODE REVIEW

File: dev-aws/finance/debt-payments-page.tf [hunk 1]
```diff
@@ -0,0 +1,54 @@
+resource "kafka_topic" "bq_staged_events" {
+  name               = "bq-staged-events"
```
Comment (tosan88) [inline dev-aws/finance/debt-payments-page.tf:2]: In theory these topics aren't defined just for the debt payments' page, except the `debt-payment-transaction` and `failed-payment-confirmation` topics. I think the others should not be put in this `.tf` file but in another

Comment (victoriaUW) [reply dev-aws/finance/debt-payments-page.tf:2]: Ok makes sense. Can the other three be in one file? Do you have a naming suggestion?

Comment (tosan88) [reply dev-aws/finance/debt-payments-page.tf:2]: Thanks! The `overdue-balance-with-payments-diffs` could go to the already existing `debt.tf` and actually the `debt-payment-transaction` and `failed-payment-confirmation` topics could go there as well. However for the others I have a hard time to come up with something, as they were [supposed to be used by other pipelines as well](https://github.com/utilitywarehouse/finance-manifests/blob/20d52fe5279d76dddc83a2a8c4c79a9b941a691b/balance-change-stager/app.yaml#L91). Maybe some generic naming, like "common.tf"? Maybe worth asking the others as well

Comment (victoriaUW) [reply dev-aws/finance/debt-payments-page.tf:2]: Thank Sandor, I will update the debt.tf and maybe @danielcopaciu has a suggestion?

Comment (danielcopaciu) [reply dev-aws/finance/debt-payments-page.tf:2]: @tosan88 @victoriaUW What do you think about creating a file for BQ only? Potentially you could have all the topics in one file but I thought the file would get too large so I decided to split them by areas.

Comment (tosan88) [reply dev-aws/finance/debt-payments-page.tf:2]: Fine by me!

**Review by tosan88 — APPROVED** [review]

**Pull request merged**